fuse_examples/classification/cmmd/runner.py

```diff
@@ -298,57 +250,37 @@ def run_eval(paths: dict, eval_common_params: dict):
 
     # run
     results = evaluator.eval(ids=None,
-                     data=os.path.join(paths["inference_dir"], eval_common_params["infer_filename"]),
+                     data=os.path.join(cfg.paths.inference_dir, cfg.infer.infer_filename),
                      metrics=metrics,
-                     output_dir=paths['eval_dir'])
+                     output_dir=cfg.paths.eval_dir)
 
     return results
-######################################
-# Run
-######################################
-
 
-if __name__ == "__main__":
-    # allocate gpus
-    if NUM_GPUS == 0:
-        TRAIN_COMMON_PARAMS['manager.train_params']['device'] = 'cpu'
+@hydra.main(config_path="conf", config_name="config")
+def main(cfg : DictConfig) -> None:
+    print(cfg)
     # uncomment if you want to use specific gpus instead of automatically looking for free ones
     force_gpus = None  # [0]
-    FuseUtilsGPU.choose_and_enable_multiple_gpus(NUM_GPUS, force_gpus=force_gpus)
+    FuseUtilsGPU.choose_and_enable_multiple_gpus(cfg.train.manager_train_params.num_gpus, force_gpus=force_gpus)
 
-    RUNNING_MODES = ['train', 'infer', 'eval']  # Options: 'train', 'infer', 'eval'
-    # Path to save model
-    root = ''
-    # Path to the stored CMMD dataset location
+    RUNNING_MODES = ['train', 'infer', 'analyze']  # Options: 'train', 'infer', 'analyze'
+    # Path to the stored dataset location
     # dataset should be download from https://wiki.cancerimagingarchive.net/pages/viewpage.action?pageId=70230508
     # download requires NBIA data retriever https://wiki.cancerimagingarchive.net/display/NBIA/Downloading+TCIA+Images
-    # put on the following in the main folder  - 
+    # put on the folliwing in the main folder  - 
     # 1. CMMD_clinicaldata_revision.csv which is a converted version of CMMD_clinicaldata_revision.xlsx 
     # 2. folder named CMMD which is the downloaded data folder
```

Comment:
I guess that all these comments now should go into the configuration file too, right?

Reply:
it's still relevant.. it's on how to download the dataset